.github/workflows/format.yaml

```
@@ -7,15 +7,22 @@ on:
   pull_request:
     branches: ["main"]
 
+permissions:
```

maybe here we set

```suggestion
permissions: {}
```

and set the permission in the job level and not in the workflow level